README.md

```diff
@@ -34,9 +36,41 @@ For alternative ways, stop by our Community Hub [stackrox.io](https://www.stackr
     + [How to Deploy](#how-to-deploy)
 * [Generating portable installers](#generating-portable-installers)
 
-## Quick Installation via Helm
+## Installation via Helm
+
+### Quick Installation
 
 StackRox offers quick installation via Helm Charts. Follow the [Helm Installation Guide](https://helm.sh/docs/intro/install/) to get `helm` CLI on your system.
+Then run the helm quick install script or proceed to section [Manual Helm Installation](#manual-helm-installation) for configuration options.
+```sh
+./scripts/quick-helm-install.sh 
+```
+The script will add the stackrox helm repository, generate an admin password, install stackrox-central-services, create an init bundle for shared secrets, and finally create one secured cluster.
+If you wish to create more secured clusters you may do so using
+```sh
+helm install -n stackrox stackrox-secured-cluster-services stackrox/stackrox-secured-cluster-services \
+  -f stackrox-init-bundle.yaml \
+  --set clusterName="$CLUSTER_NAME"
+```
```

Comment:
1. I don't think there's any need to provide this info here given that it's printed by the `quick-helm-install.sh` script.
2. This command isn't correct in general case because _additional_ secured clusters will need a central endpoint which is external for them.

I suggest to take it out from here and potentially from the script. Instead, I suggest to link to documentation that explains `stackrox/stackrox-secured-cluster-services` in greater detail (you actually have it below).

Reply:
Makes sense to me, I will do this.